website/docs/advanced/builder.md

```diff
@@ -0,0 +1,211 @@
+
+Only validators that are active will be registered against the builder. Registrations for applicable validators will be pushed at the start of the validator client and at the start of each epoch. Success of the API is not guaranteed and will try again at the start of each epoch.
+
+The beacon node must also be configured to enable builder via the `--http-mev-relay` flag.
+
+:::
+
+## 2. Beacon Node: connect to builder
+
+To use a Builder the beacon node needs to start with the following configuration:
+
+- `--http-mev-relay` flag pointed to a mev boost instance or running relay (5.Builder via Relay URL).
+
+Each Relay's URL will correspond to a specific network and will need to be chosen accordingly. i.e. running beacon node on mainnet will require the mainnet relay.
+
+### circuit breaker
+
+is a safety feature for falling back to local execution when using the Builder.
+This occurs when the builder or client using the builder endpoints encounter issues which cause missed blocks. 
+By default the circuit breaker will be triggered after 3 slots are consecutively missed or 5 slots are missed in an epoch, but this can be configured through the `--max-builder-consecutive-missed-slots` and `max-builder-epoch-missed-slots` flags.
+
+## Enable Registration Cache
+
+`--enable-registration-cache` flag will enable storage of validator registrations in a cache instead of bolt db when starting the beacon node. The cache will enable the following features.
+  - in memory storage of the validator registrations, this clears all validator registrations on restart.
+  - validator registrations will expire after 3 epochs unless sent consistently from the validator client.
+This feature solves the unintended issue of wanting some validators unregistered while maintaining mev boost on others. In the future the db used to store registrations will be removed completely and the flag will no longer be required for this feature. validator settings will be persisted on the validator client side.
+
+:::caution
+
+This feature is currently `not ready` for [Keymanager-API's](../how-prysm-works/keymanager-api.md) use as restarting the validator and beacon node (especially during upgrades) will clear the validator settings for validator registrations. 
+Current persisted values in the db will start but will not be migrated to the cache.
+
+:::
+
+### Parallel Execution
+
+`--build-block-parallel` flag on the beacon node will construct the consensus and execution portions of the beacon block in parallel to improve speed and efficiency. 
+
+### prioritizing local blocks
+
+`--local-block-value-boost` flag is a float64 value that provides an additional percentage to multiply local block value. Use builder block if: `builder_bid_value * 100 > local_block_value * (local-block-value-boost + 100)`. This will encourage your setup to use the local execution if the value earned is not above your threshold, helping to mitigate censorship concerns.
```

Comment:
```suggestion
`--local-block-value-boost` flag is a float64 value that provides an additional percentage to multiply the local block value. Use builder block if: `builder_bid_value * 100 > local_block_value * (local-block-value-boost + 100)`. This will encourage your setup to use the local execution if the value earned is not above your threshold, helping to mitigate censorship concerns.
```